jobs/keepalived/templates/config/keepalived.conf.erb

```diff
@@ -0,0 +1,46 @@
+<% internal = link("broker").p('internal') %>
+vrrp_script chk_sfnode {
+    script "/var/vcap/jobs/keepalived/bin/get_sf_broker_status.sh"
+    weight 10
+    interval 2
+    fall 2
+    rise 2
+}
+vrrp_instance VI_1 {
+    debug 2
+    interface eth0 # interface to monitor
+    <% if ( spec.bootstrap === TRUE ) %>
```

Comment:
where is this bootstrap in spec?

Reply:
spec.bootstrap: True if this instance is the first instance of its group. ==> https://bosh.io/docs/jobs/